packages/yew-macro/tests/html_macro/html-if-fail.stderr

```diff
@@ -0,0 +1,35 @@
+error: expected block after this condition
+ --> $DIR/html-if-fail.rs:4:16
+  |
+4 |     html! { if {} };
+  |                ^^
```

Comment:
Should be expecting the condition here.
The `{}` is being parsed because it is of type `syn::Expr` and then you run out of tokens which trips your first input.is_empty() check. 
I think most users would realise what the error was about, but I still don't think it's correct.